netty/src/main/java/io/grpc/netty/NettyClientTransport.java

```diff
@@ -165,14 +166,20 @@ public ClientStream newStream(MethodDescriptor<?, ?> method, Metadata headers) {
   public Runnable start(Listener transportListener) {
     lifecycleManager = new ClientTransportLifecycleManager(
```

Comment:
I see. I think move it inside `start()` is cleaner since it's only one line of code and used only once.